doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3621,6 +3621,8 @@ See also: Proof. in :ref:`TODO-7.1.4-Proofterm`.
    time the term argument of a tactic has one of its holes not fully
    resolved.
 
+   This command is deprecated. Use typeclasses or tactics-in-terms instead.
```

Comment:
I once heard you saying that tactics-in-terms was a mistake and now I see you recommend it.
What's happened?

(I personally like "the idea" tactics in terms, not quite their status today)

Reply:
Interested too by your answer to this question @ppedrot.

Reply:
Tactics-in-terms per se is OK, it's just that in the context of Ltac, it is a horrible source of hackish workarounds to the lack of expressivity of the language. So the feature is useful, but the uses are  often disgusting.